datalake/backend/endpoints/parameter_manager.py

```diff
@@ -0,0 +1,21 @@
+import os
+from enum import Enum, auto
+
+import boto3
+
+ssm_client = boto3.client("ssm")
+
+ENV = os.environ["DEPLOY_ENV"]
+
+
+class ResourceName(Enum):
+    DATASETS_TABLE_NAME = auto()
+    PROCESSING_ASSETS_TABLE_NAME = auto()
+    STORAGE_BUCKET_NAME = auto()
+    DATASET_STAGING_BUCKET_NAME = auto()
+
+
+def get_param(parameter: ResourceName):
+    path = f"/{ENV}/{parameter.value}"
+    ssm_parameter = ssm_client.get_parameter(Name=path)
+    return ssm_parameter["Parameter"]["Value"]
```

Comment:
We might want to do something like `assert ssm_parameter["Result"] == success, ssm_parameter` to get some details if the request failed.